static/styles/settings.css

```diff
@@ -908,17 +908,24 @@ input[type="checkbox"] {
 #alert_words_list {
     margin: 0;
 
+    .alert-word-information-box {
+        min-width: 98%;
+        float: left;
+    }
+
+    .edit-alert-word-buttons {
+        float: left;
+    }
+
```

Comment:
These classes don’t exist anymore.

Reply:
Right! Just removed, there were a few locations where this was. Removed now. Just pushed.